### PR TITLE
Use node 12 images

### PR DESCRIPTION
**Pull request by ericwill**
Use node 12 as a base image, or in the case of che-sidecar-node, the main image.

Signed-off-by: Eric Williams <ericwill@redhat.com>

### CODE REVIEW

File: v3/plugins/redhat/php/1.3.11/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ category: Language
 firstPublicationDate: "2020-05-26"
 spec:
   containers:
-    - image: "quay.io/eclipse/che-sidecar-php:7-fc377b6"
+    - image: "quay.io/eclipse/che-sidecar-php:7-9854327"
```
Comment (benoitf) [inline v3/plugins/redhat/php/1.3.11/meta.yaml:15]: do we know why php is requiring nodejs ?

Comment (ericwill) [reply v3/plugins/redhat/php/1.3.11/meta.yaml:15]: I'm not sure TBH, it's been there from the beginning: https://github.com/che-dockerfiles/che-sidecar-php/commit/295ec3296ce7720511e98d5f4736f0ad8028ee98

Comment (benoitf) [reply v3/plugins/redhat/php/1.3.11/meta.yaml:15]: looks like it's a leftover when we had to have nodejs for che-theia endpoint but not related directly to this PR

Comment (ericwill) [reply v3/plugins/redhat/php/1.3.11/meta.yaml:15]: Alright, I'll remove node as the base image and use an alpine image base instead

**Pull request merged**